visidata/features/layout.py

```diff
@@ -40,11 +40,16 @@ def hide_col(vd, col):
 def hide_uniform_cols(sheet):
     if len(sheet.rows) < 2:
         return
-    for col in sheet.visibleCols:
+    idx = sheet.cursorVisibleColIndex
+    for i, col in enumerate(sheet.visibleCols):
         vals = (col.getTypedValue(r) for r in sheet.rows)
         first = next(vals)
         if all(v == first for v in vals):
+            vd.status(f'hid col {col.name} with value: {repr(first)}')
```

Comment:
Let's show the display value instead of the repr of the typed value.  Also I'd like this status to say "hide {col.name} with value {val}", to avoid the irregular past-tense verb and the abbreviated and unnecessary "col".

Reply:
Okay, I've fixed that line for display value and wording. The past tense did feel out of step with other status messages.